lib/deliver/submit_for_review.rb

```diff
@@ -59,6 +47,17 @@ def select_build(options)
         raise "Could not find build to select".red
       end
 
+      loop do
+        break if build.processing == false
+
+        Helper.log.info "Waiting iTunes Connect processing... this might take a while..."
+        if (Time.now - start) > (60 * 5)
+          Helper.log.info ""
+          Helper.log.info "You can tweet: \"iTunes Connect #iosprocessingtime #{((Time.now - start) / 60).round} minutes\""
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.